docs/text_summaries.ipynb

```diff
@@ -213,24 +247,51 @@
         "\n",
         "# Using the file writer, log the text.\n",
         "with file_writer.as_default():\n",
-        "  with tf.name_scope(\"Name scope 1\"):\n",
+        "  with tf.name_scope(\"name_scope_1\"):\n",
         "    for step in range(20):\n",
-        "      tf.summary.text(\"a stream of text\", f\"Hello from step {step}\", step=step)\n",
-        "      tf.summary.text(\"another stream of text\", f\"This can be kept separate {step}\", step=step)\n",
-        "  with tf.name_scope(\"Name scope 2\"):\n",
-        "    tf.summary.text(\"just from step 0\", f\"This is an important announcement from step 0\", step=0)\n",
+        "      tf.summary.text(\"a_stream_of_text\", f\"Hello from step {step}\", step=step)\n",
+        "      tf.summary.text(\"another_stream_of_text\", f\"This can be kept separate {step}\", step=step)\n",
+        "  with tf.name_scope(\"name_scope_2\"):\n",
+        "    tf.summary.text(\"just_from_step_0\", \"This is an important announcement from step 0\", step=0)\n",
         "    "
       ]
     },
     {
       "cell_type": "code",
-      "execution_count": null,
+      "execution_count": 14,
       "metadata": {
         "id": "515199f4b547"
       },
-      "outputs": [],
+      "outputs": [
+        {
+          "data": {
+            "text/html": [
+              "\n",
+              "      <iframe id=\"tensorboard-frame-4a8120aa135ba5f2\" width=\"100%\" height=\"800\" frameborder=\"0\">\n",
+              "      </iframe>\n",
+              "      <script>\n",
+              "        (function() {\n",
+              "          const frame = document.getElementById(\"tensorboard-frame-4a8120aa135ba5f2\");\n",
+              "          const url = new URL(\"/\", window.location);\n",
+              "          const port = 6010;\n",
+              "          if (port) {\n",
+              "            url.port = port;\n",
+              "          }\n",
+              "          frame.src = url;\n",
+              "        })();\n",
+              "      </script>\n",
+              "    "
+            ],
+            "text/plain": [
+              "<IPython.core.display.HTML object>"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        }
+      ],
       "source": [
-        "%tensorboard --logdir logs"
+        "%tensorboard --logdir logs/multiple_texts --samples_per_plugin 'text=5'"
```

Comment:
Seems like a better example to set the sample count to something higher (e.g. at least 20, since that's the step count here, or maybe something like 100).  It seems a lot more likely that folks will want to increase the count than decrease it (relative to the default value of 10).

Reply:
I intentionally kept 5 here to illustrate to the user that subsampling occurred.  I suspect that this will encourage users to play with the parameter.  If we set it higher than 20, users will probably overlook it.